data/categories.txt

```diff
@@ -785,7 +785,7 @@ shop-car
 en:car shop|U+1F697|U+1F698|U+1F699
 ru:автосалон|магазин
 uk:автосалон|магазин
-de:Auto-Shop|Laden
+de:Autohaus
```

Comment:
Alex Zolotarev schrieb
> 
> Is "Geschäft" better as a generic "shop" or "shops" than "Laden"?
> What is the difference?

"Laden" is usually used for smaller retail businesses, rarely (if all)
for supermarkets/discounters and I'd doubt that people in Germany use it
at all for car dealers.
To describe "Geschäft" quite vaguely: A place where you get $stuff (be
it a cleaning for your tuxedo, a car or a new axe) for money